modules/ROOT/pages/dataweave-type-reuse-avro-schema.adoc

```diff
@@ -0,0 +1,150 @@
+= Reusing Types from an Avro Schema
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+An Avro schema is a formal specification that defines the structure and data types for records stored in the Apache Avro format, which is a data serialization system commonly used in big data systems like Apache Hadoop and Apache Kafka. It ensures that data written in Avro can be easily understood and processed across different systems, regardless of the programming languages or platforms involved.
```

Comment:
Is the format the data serialization system? 

Suggest breaking first sentence into two, since it's so long.

Reply:
Is the format the data serialization system?
> yes, the avro format is a serialization system

In the second sentence, does "it" refer to the schema? Maybe say "The Avro Schema ensures..."?
> Yes, it refers to the schema

Please suggest any edits you wish as commits as I'm certain of making it wrong